.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "submodules/api-common-protos"]
 	path = submodules/api-common-protos
-	url = https://github.com/googleapis/api-common-protos.git
+	url = https://github.com/googleapis/googleapis.git
```

Comment:
So, this change of URL can't be done without also changing the Python dependencies, because we are using a package providing the built `api-common-protos`.

We need to find the package that provides the whole `googleapis` repo instead.